README.md

```diff
@@ -1,3 +1,146 @@
 # Work in progress
 
-This repo is not ready for consumption, and is under heavy development
\ No newline at end of file
+This repo is not ready for consumption, and is under heavy development. Right now the Rust core bindings work only for Apple Silicon systems.
+
+## Development Prerequisites
+
+### Hermit
+
+This project uses hermit to manage tooling like the Rust compiler, Java Development Kit and Maven project management system.
+See [this page](https://cashapp.github.io/hermit/usage/get-started/) to set up Hermit on your machine - make sure to
+download the open source build and activate it for the project.
+
+Once you've installed Hermit and before running builds on this repo,
+run from the root:
+
+```shell
+source ./bin/activate-hermit
+```
```

Comment:
is this the 'default hermit' readme? Isn't it set up so you just cd in a directory and hermit runs this automagically?

Reply:
That automation occurs via [shell hooks](https://cashapp.github.io/hermit/usage/shell/) -- might be we should make not of it here in the README for dev's who are willing to setup that automation